index-nl.html

```diff
@@ -62,25 +62,25 @@
 
 <h2>De minder korte versie</h2>
 
-<p>Onder lastig vallen verstaan wij beledigende mondelinge opmerkingen over geslacht, seksuele oriëntatie, handicap, uiterlijk, lichaamsgrootte, ras, geloof, seksueel getinte afbeeldigen in openbare ruimten, bewuste intimidatie, stalken, achtervolgen, nemen van foto's of andere opnames, volhardend interumperen van praatjes of andere activiteiten, ongepast fysiek contact en ongewenste seksuele aandacht.</p>
+<p>Onder lastig vallen verstaan wij beledigende mondelinge opmerkingen over geslacht, seksuele oriëntatie, handicap, uiterlijk, lichaamsgrootte, ras, geloof, seksueel getinte afbeeldingen in openbare ruimten, bewuste intimidatie, stalken, achtervolgen, nemen van foto's of andere opnames, volhardend interrumperen van praatjes of andere activiteiten, ongepast fysiek contact en ongewenste seksuele aandacht.</p>
 
 <p>Als deelnemers worden gevraagd op te houden met lastig gedrag, wordt verwacht dat zij dit meteen doen.</p>
 
 <p>Sponsors dienen zich ook aan de gedragscode te houden. In het bijzonder dienen zij geen gebruik te maken van seksueel getinte beelden, activiteiten of ander materiaal. Standhouders, waaronder vrijwilligers, mogen geen seksueel getinte kleding/uniformen/kostuums dragen of in andere zin een seksueel milieu cre&euml;ren.</p>
 
 <p>Als een deelnemer zich ongepast gedraagt mag het organiserend comit&eacute; gepaste actie ondernemen. Zij mogen hierbij de deelnemer waarschuwen zelfs of de toegang tot de conferentie ontzeggen zonder schadeloosstelling.</p>
 
-<p>Als je wordt lastiggevallen, of merkt dat iemand anders wordt lastiggevallen, of je hebt andere zorgen, meld je dan meteen bij een van de conferentiemedewerkers. Conferentiemedewerkers zijn te herkennen aan hun t-shirts.</p>
+<p>Als je wordt lastiggevallen, of merkt dat iemand anders wordt lastiggevallen, of je hebt andere zorgen, meld je dan meteen bij een van de conferentiemedewerkers. Conferentiemedewerkers zijn te herkennen aan hun T-shirts.</p>
 
-<p>Conferentiemedewerkers zullen niet aarzelen om de beveiling van de locatie in te schakelen, de politie te bellen, je te begeleiden of anderszins van dienst te zijn wanneer je wordt lastig gevallen. Wij waarderen je aanwezigheid.</p>
+<p>Conferentiemedewerkers zullen niet aarzelen om de beveiliging van de locatie in te schakelen, de politie te bellen, je te begeleiden of anderszins van dienst te zijn wanneer je wordt lastiggevallen. Wij waarderen je aanwezigheid.</p>
```

Comment:
“lastiggevallen” was suggested by Google Translate as “lastig gevallen”, double-checked with reverso.net which indeed supported the space removal.